src/telegram.ts

```diff
@@ -1,10 +1,32 @@
-import * as tg from './core/types/typegram'
+import * as tg from 'typegram'
 import * as tt from './telegram-types'
-import ApiClient from './core/network/client'
+import { TT } from './telegram-types'
+import { Client, ClientOptions, createApi, type Opts } from '@telegraf/client'
 import { isAbsolute } from 'path'
 import { URL } from 'url'
+import { TelegramError } from './index'
+
+export class Telegram {
+  client: Readonly<Client>
```

Comment:
It must be `#private`, otherwise every middleware can access `ctx.telegram.client.context`.

Reply:
`Telegraf` must store `Client`.